spec/models/custom/poll/question_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe Poll::Question, type: :model do
+  let(:poll_question) { build(:poll_question) }
+
+  describe "#single_choice?" do
+    it "returns false when question has no answers" do
+      poll_question = create(:poll_question)
+
+      expect(poll_question.single_choice?).to be_falsy
+    end
+
+    it "returns false when question has no answers" do
```

Comment:
I guess there's a typo here:

```suggestion
    it "returns true when the question has answers" do
```